test/fixture/types.ts

```diff
@@ -1,6 +1,7 @@
 import { expectTypeOf } from "expect-type";
 import { describe, it } from "vitest";
 import { EventHandler, EventHandlerRequest, defineEventHandler } from "h3";
+// @ts-ignore
```

Comment:
Tests failed on me here, added ts-ignore. Didn't know if I had to run some sort of `prepare` to fix this issue. Remove if unnecessary